### ISSUE TITLE
Please check if the title of the weblink module shows up in a google article.

### ISSUE THREAD

**Issue by rambro** [joomla-extensions/weblinks#503]
![Getting a show title error](https://user-images.githubusercontent.com/4724475/177191541-6a1bad0d-929c-4683-97ab-c757fb5a41d5.jpg)

**Comment by chmst** [comment]
I don't understand your issue. What do you mean with Google Article? 
If you are missing the headline of the module, check if the module style is html5.

**Comment by rambro** [comment]
Dear Chmst,

OK, the html5 tag (after I figured it out) worked for the weblink category headings. However, I am still getting the link and then the description of the link appearing to the side of the link instead of  under the link. How would I get the description of the link to appear underneath the weblink?

**Comment by rambro** [comment]
Hi chmst,

Thanks for getting back to me with the info. I got it to work. I made changes to line 42 and 103 of the code. I am going to attach some pictures.
![changes made the weblinks module via override_line 42](https://user-images.githubusercontent.com/4724475/180468342-744c6032-d139-42f1-accc-3688be05031f.jpg)
![changes made to the weblinks module via override_line 103](https://user-images.githubusercontent.com/4724475/180468389-496f1b1e-db25-4e0a-bf36-3bfda124c9f0.jpg)
![website page after override page 1 of 2](https://user-images.githubusercontent.com/4724475/180468461-9d3e6f64-a821-45fb-9c73-3aa3632c4c28.jpg)
![website page after override page 2 of 2](https://user-images.githubusercontent.com/4724475/180468483-531a3ab4-afbf-4404-a7f9-aae26ea2fbd7.jpg)
